### DISCUSSION TITLE
Add option to set password for user (not for config file)

### DISCUSSION THREAD

**Discussion by vitonsky** [angristan/openvpn-install#1112] [Ideas]
**Checklist**

- [x] I read the [README](https://github.com/angristan/openvpn-install/blob/master/README.md)
- [x] I read the [FAQ](https://github.com/angristan/openvpn-install/blob/master/FAQ.md)
- [x] I searched the [issues](https://github.com/angristan/openvpn-install/issues?q=is%3Aissue+)
- [x] My issue is about the script, and not OpenVPN itself

**Is your feature request related to a problem? Please describe.**
I need to give access to VPN to my friends, but sometime i want disable access for them and enable after some time.
I need to use passwords, to give them one ovpn file and change password when i want and send them new password later.
When i use this script, i can't setup password for user.

I found issue about this https://github.com/angristan/openvpn-install/issues/200 but its closed.
I can't use this script without this feature due to security reasons.

**Describe the solution you'd like**
Implement option to set password for each user. Not for config, but for user. For example, to when i try to use ovpn file in kde connector, i'll have to enter username and password.

**Describe alternatives you've considered**
At this time i can only recreate users each time, it is unacceptable and make this script useless for my purposes

**Additional context**
This feature would make configurations more flexible

**Comment by benmotyka** [comment]
+1

**Comment by ja49619** [comment]
Do you understand how you can set a password for a client?
